plugins/modules/azure_rm_privateendpoint_info.py

```diff
@@ -116,8 +116,49 @@
             description:
                 - The resource id of the private endpoint to connect.
             returned: always
+            type: complex
+            returned: always
+            contains:
+                id:
+                    description:
+                        - The resource id of the private endpoint to connect.
+                    returned: always
+                    type: str
+                name:
+                    description:
+                        - The name of the private endpoint connection.
+                    returned: always
+                    type: str
+                connection_state:
+                    description:
+                        - State details of endpoint connection
+                    type: complex
+                    returned: always
+                    contains:
+                        description:
+                            description:
+                                - The reason for approval/rejection of the connection.
+                            returned: always
+                            type: str
+                            sample: "Auto Approved"
+                        status:
+                            description:
+                                - Indicates whether the connection has been Approved/Rejected/Removed by the owner of the service.
+                            returned: always
+                            type: str
+                            sample: Approved
+                        actions_required:
+                            description:
+                                - A message indicating if changes on the service provider require any updates on the consumer.
+                            type: str
+                            returned: always
+                            sample: "This is action_required string"
+        group_ids:
```

Comment:
Then move eight Spaces。
```suggestion
                group_ids:
                    description:
                        - List of group_ids associated with private endpoint
                    returned: always
                    type: list
                    sample: [""postgresqlServer"]
``